config/coordinator.sample.json

```diff
@@ -13,34 +13,51 @@
     "UpdateIntervalMs": 2000,
     "Queries": [
       {
-        "Description": "Average request duration (old)",
+        "Description": "Average server request duration (old)",
         "Query": "rate(mattermost_http_request_duration_seconds_sum[1m])/rate(mattermost_http_request_duration_seconds_count[1m])",
         "Threshold": 0.1,
         "Alert": true
       },
       {
-        "Description": "Average request duration",
+        "Description": "Average server request duration",
         "Query": "sum(rate(mattermost_api_time_sum[1m]))/sum(rate(mattermost_api_time_count[1m]))",
         "Threshold": 0.1,
         "Alert": true
       },
       {
-        "Description": "Rate of HTTP 5xx client errors",
-        "Query": "sum(rate(loadtest_http_errors_total{status_code=~\"5..\"}[1m]))",
-        "Threshold": 0.01,
+        "Description": "Percentage of HTTP 5xx server errors",
+        "Query": "(sum(rate(mattermost_api_time_count{status_code=~\"5..\"}[1m]))/sum(rate(mattermost_api_time_count[1m])))*100",
+        "Threshold": 0.025,
```

Comment:
I've changed this to be a percentage of errors over the total number of requests to make it scale better.
This helps with false positives and lets us define an acceptable tolerance to errors.

Reply:
Much better approach 👍